atmos.yaml

```diff
@@ -383,6 +383,30 @@ settings:
       color: "${colors.muted}"
       italic: true
 
+
+  docs:
+    generate:
```

Comment:
finally, I'd prefer if the default option `atmos generate docs` would build all docs options without subcommands. For now that would only be `readme`